easybuild/easyconfigs/p/Python/Python-3.11.5-GCCcore-13.2.0.eb

```diff
@@ -10,8 +10,16 @@ toolchainopts = {'pic': True}
 
 source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
 sources = [SOURCE_TGZ]
-checksums = ['a12a0a013a30b846c786c010f2c19dd36b7298d888f7c4bd1581d90ce18b5e58']
+patches_filter_ld_library_path = ['Python-3.11.5-custom-ctypes.patch']
+checksums_filter_ld_library_path = [
+    {'Python-3.11.5-custom-ctypes.patch': '8ae3efc47dd202b600de8249df31b8be252814933b8339ba42fe5c2e0d9adb14'}
+]
+checksums = [
+    {'Python-3.11.5.tgz': 'a12a0a013a30b846c786c010f2c19dd36b7298d888f7c4bd1581d90ce18b5e58'},
+]
 
+# Like patches, but these will only be applied if EasyBuild is configured to filter LD_LIBRARY_PATH
```

Comment:
This comment should be moved right before the `patches_filter_ld_library_path` keyword is defined